bin/v-update-sys-hestia-git

```diff
@@ -20,23 +20,10 @@ source /etc/hestiacp/hestia.conf
 source $HESTIA/func/main.sh
 # load config file
 source_conf "$HESTIA/conf/hestia.conf"
-# define NodeJS version for download (required for building JS/CSS)
-nodejs_ver="20"
 
 # Perform verification if read-only mode is enabled
 check_hestia_demo_mode
 
-# Detect and install NodeJS if necessary
-if [ -z $(which "node") ]; then
-	read -p "NodeJS not found. Install now to proceed? [Y/n] " answer
-	if [ "$answer" = 'y' ] || [ "$answer" = 'Y' ]; then
-		curl -fsSL "https://deb.nodesource.com/setup_$nodejs_ver.x" | bash - &&\
-		sudo apt-get install -y nodejs
```

Comment:
We still might need to run nodejs in rare occasions .... 

So it probably doesn't hurt to keep that part present for now ...

Reply:
Yeah I ended up adding it back, but without the nodesource setup. Both the install and upgrade script for 1.9.0 install nodesource.